roles/st2/tasks/user.yml

```diff
@@ -51,3 +50,10 @@
     option: ssh_key_file
     value: "{{ _user.ssh_key_file }}"
     backup: yes
+
+- name: Disable requiretty
+  become: yes
+  replace:
+    dest: "/etc/sudoers"
+    regexp: '^Defaults\s+\+?requiretty'
+    replace: '# Defaults requiretty'
```

Comment:
Additionally, please take a look at https://github.com/stackstorm/ansible-st2#variables `st2_system_user_in_sudoers` var and https://github.com/StackStorm/ansible-st2/blob/156a0844f83a37bd601420be2a10cc85b64bc34e/roles/st2/tasks/user.yml#L32 usage.

So we need to change `sudoers` only when `st2_system_user_in_sudoers: yes`